### DISCUSSION TITLE
Playlist jingle

### DISCUSSION THREAD

**Discussion by Rafaelrds2017** [savonet/liquidsoap#2009] [Q&A]
** Is the request feature related to an issue? Please describe.**

A: Playlist does not play all audio at once.

A clear and concise description of what the problem is. For example. I always get frustrated when [...]

A: The Playlist has 5 commercials, w should play all 5 commercials in sequence, and at the scheduled time. but that doesn't happen. Sometimes it plays only the first audio.

** Describe the solution you would like **
A clear, concise description of what you want to happen.

A: The Commercial Playlist or Jingle must obey the option to play the entire Playlist at the scheduled time.

** Describe the alternatives you considered **
A clear and concise description of any alternative solutions or resources you have considered.

** Additional context **
Add any other context or screenshots about the resource request here.

**Comment by smimram** [comment]
I cannot understand this bug report.

**Comment by Rafaelrds2017** [comment]
Playlist does not play all audio at once.
The Playlist has 5 commercials, w should play all 5 commercials in sequence, and at the scheduled time. but that doesn't happen. Sometimes it plays only the first audio.
The Commercial Playlist or Jingle must obey the option to play the entire Playlist at the scheduled time.
can someone help me with this? will it be a bug or improper configuration?

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Rafaelrds2017** [comment]
Example

I try to create a playlist of jingle commercials.

this Playlist contains 5 audios.

and I want this Playlist to play all audios at the scheduled time.

ie play the 5 commercials/jingle


but in practice it doesn't play all the audios. sometimes it plays only the first audio only.


already configured to reproduce the entire Playlist at once at the scheduled time.

already configured to merge the Playlist to play as a single track

but it does not work


sorry if you don't understand. I do not speak english.


________________________________

**Reply by smimram** [reply]
It would help very much if you could provide the script you are using so that we can have a look.

**Comment by Rafaelrds2017** [comment]
I use azuracast. I'll check

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
